stock_available_to_promise_release/models/stock_move.py

```diff
@@ -363,6 +369,23 @@ def _run_stock_rule(self):
 
         return True
 
+    def _release_get_deadline(self, date_planned):
+        prep_time = self.env.company.stock_release_max_prep_time
+        date_deadline = fields.Datetime.add(date_planned, minutes=prep_time)
+        return date_deadline
```

Comment:
Here it is not a date deadline I guess, but a scheduled/planned date which is computed:
```suggestion
    def _release_get_date_planned(self, date_planned):
        prep_time = self.env.company.stock_release_max_prep_time
        new_date_planned = fields.Datetime.add(date_planned, minutes=prep_time)
        return new_date_deadline
```
The date deadline is when the goods are shipped to the destination location (with the time needed by the carrier let's say). If a commitment date is set on a SO, this is the date deadline of the moves.
Note: we worked a lot on this PR to handle a lot of use cases regarding time windows etc: https://github.com/OCA/sale-workflow/pull/1968/